### DISCUSSION TITLE
DER encoded cert and P12

### DISCUSSION THREAD

**Discussion by szcsdev** [matteocorti/check_ssl_cert#253] [Q&A] [answered]
Hi There!

Your check is a very good idea, but i can't check certs with DER format, and P12.
I get the message " SSL_CERT CRITICAL /cert path/cert name.p12: is not a valid certificate file "
Please help me what is wrong.
Thank you

**Comment by matteocorti** [comment]
Hi can you please send the debugging output? I'll try to implement the missing conversion next week

**Comment by matteocorti** [comment, marked as answer]
I created a new issue #254
Please add any information to the issue
